crates/orderbook/Cargo.toml

```diff
@@ -52,3 +52,4 @@ web3 = { version = "0.18", default-features = false }
 secp256k1 = "0.21"
 mockall = "0.11"
 testlib = { path = "../testlib" }
+tokio = { version = "1.15", features = ["test-util"] }
```

Comment:
Aren't we on `tokio v1.17` now?

Reply:
I copied the version from another line. If tokio 1.17 is the newest version then we're probably already using it in the lock file since it is semver compatible.